services/distribution/src/test/resources/application.yaml

```diff
@@ -65,6 +65,20 @@ services:
       min-ios: ${MIN_IOS_VERSION:0.5.0}
       latest-android: ${LATEST_IOS_VERSION:1.0.4}
       min-android: ${MIN_ANDROID_VERSION:1.0.4}
+    app-config-parameters:
+      ios-key-download-parameters:
+        cached-day-packages-to-update-on-etag-mismatch: ${KEY_DOWNLOAD_INVALIDATED_DAY_PACKAGES:[]}
```

Comment:
For the test package resources.yaml, it might make sense not to rely on the environment variables but use the expected value only, e.g.:
cached-day-packages-to-update-on-etag-mismatch: []
This prevents failing tests due to a different system configuration